scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-gating-template.yaml

```diff
@@ -1,5 +1,5 @@
 - job-template:
-    name: 'cloud-mkcloud{version}-gating'
+    name: 'cloud-mkcloud{version}-{arch}-gating'
```

Comment:
hmm, why do we want to do gating based on architecture? the sources are shared across architectures